services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/publish/LocalFile.java

```diff
@@ -29,7 +29,7 @@
   /**
    * the checksum of this file.
    */
-  private String checksum = "";
+  private String checksum;
```

Comment:
Why did you change this? The way it was ensured avoiding NPEs...

Reply:
I changed it because it seemed unnecessary, and was a recommendation Intellij gave as well. Now that you mention NPE, makes more sense to have it initialized.